### ISSUE TITLE
When I added pointNN to my network, not only did the speed of the network slow down, but the accuracy also decreased!

### ISSUE THREAD

**Issue by ifiability** [ZrrSkywalker/Point-NN#37]
May I ask, where did you insert pointNN as a plug and play module, and how did the final feature fusion work?

**Comment by SZUguigui** [comment]
of course, i insert it into GPAR,which can find in git too. I still try to train the network after i insert pointNN, so i can't tell u how i work, sorry : (
